packages/cli/test/e2e/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "e2e",
+  "description": "",
+  "version": "0.0.0",
+  "scripts": {
+    "cypress:open": "cypress open",
+    "cypress:run": "cypress run"
+  },
+  "private": true,
+  "dependencies": {},
+  "devDependencies": {
+    "cypress": "^6.4.0"
+  }
+}
```

Comment:
Ok, yes, one thing before merging:

- Move `cypress` dependency to the very root package.json of the repo. 
- Move these e2e test scripts to `packages/cli/package.json`
- Remove this package.json

Reply:
https://github.com/blitz-js/blitz/pull/1846/commits/de466050fa31458c6e4835cd5f4a71cd00b09158

If I move these e2e test scripts to packages/cli/package.json, I have to move `cypress` folder and `cypress.json`. (`packages/cli/test/e2e/cypress/`  => `packages/cli/cypress`).
Is it Ok?

Reply:
https://github.com/blitz-js/blitz/pull/1846/commits/8dda5cc7f3bbd1af0ab15f5563076f8b83f719a4
I resolved this problem by setting `cypress.json`